### ISSUE TITLE
Can't seem to recieve intent

### ISSUE THREAD

**Issue by dfcarpenter** [darryncampbell/react-native-datawedge-intents#23]
To start thanks for the great library.

- Zebra TC26 - Android 10 Build# 10-16-10.00-QG-U81-STD-HEL-04
- React Native 0.66
- Datawedge Intents 0.1.8
- Not using Hermes

I'm using a Zebra TC26 and trying to write a small custom barcode scanning app to use in a private CBRS network and I have run into some issues. Basically I can't get it to receive the intents and I am not sure what is going on. Your React Native sample app seems to work as intended but the exact same app logic on RN0.66 does not work. I'm not entirely sure it's just an issue of RN0.66 or perhaps something else. I would like to use the latest version of RN if possible ( along with React v17+ ) because of peer dependencies of a few important libraries like React Navigation along with some others required by the project.

To demonstrate what I am talking about I created a new react native project and basically used the same libraries you have ( with some using updated versions ). 

https://github.com/dfcarpenter/rn066datawedge

This will not receive intent data from scan being toggled at least on the TC26

I took some screenshots of the logs coming from your exact react-native Datawedge sample project running RN 0.64
![zebra-tc26-rn064](https://user-images.githubusercontent.com/4450286/139506666-dd1fa2f2-c5d0-4d1a-97cb-e3fcd5310f60.png)

And this does work. Below is a screenshot of the logs coming from the 0.66 version running similar code ( with a few additions like mobx so this is why I also created a fresh project without the other libraries and this also doesn't receive intents

![zebra-tc26-RN066](https://user-images.githubusercontent.com/4450286/139506815-67a716f9-c4d8-45b9-b087-e0b4506d83e6.png)
Basically a press of the scan fires off the intent but never receives anything back. Successive presses of the scan button fires off the same thing but never receives anything.

Not sure if you have any insights here or thoughts on what what I would need to change but any feedback would be welcome.

**Comment by darryncampbell** [comment]
Hi, please change this line: https://github.com/dfcarpenter/rn066datawedge/blob/master/App.js#L187 to `"PACKAGE_NAME": "com.rn066datawedge"`.  The DataWedge profile is being created successfully but is not being applied when your app is in the foreground because you still have the old demo app associated with it.
